File: omninavi/src/main/java/com/omni/omninavi/omninavi/model/CommonResponse.java
package com.omni.omninavi.omninavi.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 * Created by wiliiamwang on 31/10/2017.
 */

public class CommonResponse implements Serializable {

    @SerializedName("result")
    private String result;
    @SerializedName("error_message")
    private String errorMessage;
    @SerializedName("data")
    private Object[] data;

    public String getResult() {
        return result;
    }

    public void setResult(String result) {
        this.result = result;
    }

    public String getErrorMessage() {
        return errorMessage;
    }

    public void setErrorMessage(String errorMessage) {
        this.errorMessage = errorMessage;
    }

    public Object[] getData() {
        return data;
    }

    public void setData(Object[] data) {
        this.data = data;
    }
}

File: navisdk/src/main/java/com/omni/navisdk/module/CommonResponse.kt
package com.omni.navisdk.module

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class CommonResponse(
        @SerializedName("result") val result: String,
        @SerializedName("error_message") val errorMessage: String,
        @SerializedName("data") val data: Any
)